### PR TITLE
Move Table to shared UI library

### PR DESCRIPTION
**Pull request by carolinemodic**
Moves the election-manager version of Table to the shared ui directory. Does NOT migrate bmd/bsd to use this shared component as their local versions are slightly different. 

Unblocks moving ContestSummary to ui which is necessary for: https://zube.io/votingworks/vxsuite/c/3563

### CODE REVIEW

**Comment by carolinemodic** [comment]
@beausmith Can you help me understand why the snapshot is updating in the precinct-scanner test that is failing on circle CI? The autogenerated id for the classname is getting changed, but I didn't change anything related to that component. It uses other components from the ui library (like Prose) but nothing that adding Table should impact, does adding something to the UI library cause the other components to re-trigger those class name generators somehow?

File: libs/ui/src/Table.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ interface Props {
 
 const borderColor = 'rgb(194, 200, 203)'
 
-const Table = styled.table<Props>`
+export const Table = styled.table<Props>`
```
Comment (eventualbuddha) [inline libs/ui/src/Table.tsx:10]: Also rename `Props` to `TableProps`?

**Review by eventualbuddha — APPROVED** [review]

**Comment by eventualbuddha** [comment]
> …does adding something to the UI library cause the other components to re-trigger those class name generators somehow?

I think this is what's happening. @beausmith probably knows more, but I don't think this is worrying.

**Comment by carolinemodic** [comment]
Unfortunate :(, will add a commit updating the snapshot.

**Review by beausmith — APPROVED** [review]
@carolinemodic - LGTM. I don't know for certain, but I assume that the change to the [Table](https://github.com/votingworks/vxsuite/pull/821/files#diff-5b3c2bb427b051d531d1ed6eb4a29368d380c1cc34c5db3cf6e31dcc3b7054a8) component are causing the class name changes. As long as the css isn't also changing I think we're good. Wouldn't hurt to load and view the screen where this component is rendered.

**Comment by carolinemodic** [comment]
Checked locally and didn't notice any visual changes in ElectionInfoBar for precinct-scanner

**Pull request merged**